### PR TITLE
FIX: missing keyword only arguments on example

### PR DESCRIPTION
**Pull request by skoudoro**
This short PR fix some warnings on our tutorials.

All warnings were coming from keyword-only arguments feature recently added

### CODE REVIEW

File: doc/examples/syn_registration_2d.py [hunk 1]
```diff
@@ -91,13 +91,13 @@
...
 
-warped_moving = mapping.transform(moving, "linear")
+warped_moving = mapping.transform(moving, interpolation="linear")
 regtools.overlay_images(
     static,
     warped_moving,
     title0="Static",
     title_mid="Overlay",
-    title_1="Warped moving",
+    title1="Warped moving",
```
Comment (arokem) [inline doc/examples/syn_registration_2d.py:100]: Is this a recent change to the api of this function?

**Review by arokem — APPROVED** [review]

**Comment by arokem** [comment]
Overall looks good to me but just one small question

File: doc/examples/syn_registration_2d.py (hunk 1 above)
Comment (skoudoro) [reply doc/examples/syn_registration_2d.py:100]: no, it was a typo not detected on a previous PR

**Comment by jhlegarreta** [comment]
BTW, how come these have not been captured by the warning decorators @deka27 ?

**Comment by jhlegarreta** [comment]
> they are capture, we just do not run the examples in our CI. I try to run each one of them to prepare the reease

If the reason is that they take too long, maybe we could run them only once a week.

**Comment by skoudoro** [comment]
> If the reason is that they take too long, maybe we could run them only once a week.

github action limit is 6h. We need to try but I believe it take more than 6h to run all examples. A strategy could be to split them in different build. We just need to block time to put that in place

**Pull request merged**